draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -144,17 +144,16 @@ The protocol described here bases its design on the following protocol requireme
 # Selection of DNS API Server {#selection}
 
 A DNS API client uses configuration to select the URI, and thus the
-DNS API server, used for resolution. [RFC2818] defines how HTTPS
-verifies the server's identity.
+DNS API server, that is to be used for resolution. [RFC2818] defines how HTTPS
+verifies the DNS API server's identity.
 
-A client MUST NOT use a different URI simply because it was discovered
-outside of configuration, or because it offers an unsolicited response
-that appears to be a valid answer to a DNS query. Specifically, this
+A DNS API client MUST NOT use a different URI simply because it was discovered
+outside of the client's configuration, or because a DNS API server offers an unsolicited response
```

Comment:
server rather than DNS API server (part of the issue here is that it may not be compliant with this doc)